### PR TITLE
Restructure pep8 tests

### PR DESCRIPTION
**Pull request by spisarski**
Here is the Big Bang approach. I've executed the script with the following arguments from the project's root directory without error:

python iaas_launch.py -f conf/hosts.yaml -p
python iaas_launch.py -f conf/hosts.yaml -pc

Can someone verify that the script is functioning as expected?

### CODE REVIEW

**Comment by rainihixon** [comment]
While testing this, I am finding that the files that are supposed to be transferred to /var/lib/tftpboot are not being copied.

**Comment by rainihixon** [comment]
I am not allowed to change the install.md file...that said:
In section 4.1, step 1:
mkdir images
cd images
change to:
mkdir -p packages/images
cd packages/images

**Comment by rainihixon** [comment]
When I try to run the static network config step, I get the following output:

root@lab1-build:/home/rlevensalor/snaps-boot# python iaas_launch.py -f ../snaps-boot-master/snaps-boot/hosts.yaml -s
CWD_IAAS path is exported implicitly
/home/rlevensalor/snaps-boot/snaps_boot/
INFO:iaas_launch:Launching Operation Starts ........
===========================Current Exported Relevant Path==================================
INFO:iaas_launch:export CWD_IAAS=/home/rlevensalor/snaps-boot/
INFO:file_utils:Configuration Loaded
INFO:file_utils:Closing config file
INFO:iaas_launch:Read configuration file - ../snaps-boot-master/snaps-boot/hosts.yaml
INFO:iaas_launch:Read & Validate functionality for Hardware Provisioning - staticIPConfigure
AAAAA
HOSTS---------------
[{'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.22', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.22', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.22', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.22', 'name': 'Controller'}, {'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.23', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.23', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.23', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.23', 'name': 'Compute0'}, {'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.24', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.24', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.24', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.24', 'name': 'Compute1'}, {'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.25', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.25', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.25', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.25', 'name': 'Compute2'}, {'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.26', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.26', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.26', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.26', 'name': 'Compute3'}]
{'interfaces': [{'dn': '', 'iface': 'eno1', 'name': 'Admin_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.101.22', 'type': 'management', 'gateway': '0.0.0.0'}, {'dn': '', 'iface': 'eno2', 'name': 'Tenant_Interface', 'netmask': '255.255.255.0', 'dns': '', 'address': '10.197.102.22', 'type': 'tenant', 'gateway': '0.0.0.0'}, {'dn': 'cablelabs.com', 'iface': 'eno3', 'name': 'External_Interface', 'netmask': '255.255.255.0', 'dns': '8.8.8.8', 'address': '10.197.103.22', 'type': 'data', 'gateway': '10.197.103.1'}], 'access_ip': '10.197.101.22', 'name': 'Controller'}
/root/.ssh/id_rsa already exists.
Overwrite (y/n)? /usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/root/.ssh/id_rsa.pub"
/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed

/usr/bin/ssh-copy-id: WARNING: All keys were skipped because they already exist on the remote system.
		(if you think this is a mistake, you may want to use -f option)

{'dn': '', 'https_proxy': '', 'netmask': '255.255.255.0', 'iface': 'eno1', 'target': '10.197.101.22', 'dns': '', 'address': '10.197.101.22', 'http_proxy': 'http://10.197.101.20:3142', 'type': 'management', 'gateway': '0.0.0.0'}
Traceback (most recent call last):
  File "iaas_launch.py", line 155, in <module>
    main(args)
  File "iaas_launch.py", line 80, in main
    __read_hw_config(config, "staticIPConfigure")
  File "iaas_launch.py", line 39, in __read_hw_config
    pxe_utils.__main(config, operation)
  File "/home/rlevensalor/snaps-boot/snaps_boot/provision/hardware/pxe_utils.py", line 54, in __main
    __static_ip_configure(static_dict, proxy_dict)
  File "/home/rlevensalor/snaps-boot/snaps_boot/provision/hardware/pxe_utils.py", line 560, in __static_ip_configure
    'dn': dn})
  File "/home/rlevensalor/snaps-boot/snaps_boot/ansible_p/ansible_utils/ansible_playbook_launcher.py", line 39, in __launch_ansible_playbook
    raise Exception('Requested playbook is not found - ' + playbook)
Exception: Requested playbook is not found - ansible_p/commission/hardware/playbooks/setIPConfig.yaml

**Comment by rainihixon** [comment]
I am able to get the static config to work if I create a link to ansible_p:

ln -s snaps_boot/ansible_p ./

File: doc/source/install/install.md [hunk 1]
```diff
@@ -345,7 +345,7 @@ State should be active running.
 Run `iaas_launch.py` as shown below:
 
 ```
-sudo -i python $PWD/iaas_launch.py -f $PWD/hosts.yaml -b
+sudo -i python iaas_launch.py -f $PWD/conf/pxe_cluster/hosts.yaml -b
```
Comment (RandyLevensalor) [inline doc/source/install/install.md]: I think that the $PWD needs to be added back for this to work with sudo -i

File: doc/source/install/install.md [hunk 2]
```diff
@@ -361,7 +361,7 @@ Execute this step only if static IPs to be assigned to host machines.
 
 Run `iaas_launch.py` as shown below:
 ```
-sudo -i python $PWD/iaas_launch.py -f $PWD/hosts.yaml -s
+sudo -i python iaas_launch.py -f $PWD/conf/pxe_cluster/hosts.yaml -s
```
Comment (RandyLevensalor) [inline doc/source/install/install.md]: I think that the $PWD needs to be added back for this to work with sudo -i

**Comment by RandyLevensalor** [comment]
Linked to issue #48

**Pull request merged**